src/views/Pages/index.js

```diff
@@ -0,0 +1,7 @@
+import Login from './Login';
+import Page404 from './Page404';
+import Register from './Register';
+
+export {
+  Login, Page404, Register
+};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)